var/spack/repos/builtin/packages/parquet-cpp/package.py

```diff
@@ -14,7 +14,11 @@ class ParquetCpp(CMakePackage):
 
     license("Apache-2.0")
 
-    version("1.4.0", sha256="52899be6c9dc49a14976d4ad84597243696c3fa2882e5c802b56e912bfbcc7ce")
+    version(
+        "1.4.0",
+        sha256="52899be6c9dc49a14976d4ad84597243696c3fa2882e5c802b56e912bfbcc7ce",
+        deprecated=True,
```

Comment:
Repo is archived: https://github.com/apache/parquet-cpp

Also can't be built with Apple Clang 17 and was only used by the GDAL package, which now suggests to directly use arrow instead.